src/pykeen/nn/emb.py

```diff
@@ -986,6 +1076,20 @@ def __init__(
         # create token representations
         # normal relations + inverse relations + padding
         total_num_tokens = 2 * triples_factory.real_num_relations + 1
+
+        # resolve anchor node selection
+        anchor_selection = anchor_selection_resolver.make_safe(anchor_selection, pos_kwargs=anchor_selection_kwargs)
+
+        # TODO: mix
```

Comment:
??

Reply:
https://github.com/migalkin/NodePiece/blob/a2189801657d41797a3db99c38f071a6468be2c5/lp_rp/nodepiece_tokenizer.py#L36

the official implementation allows for a mixture of anchor node selection strategies